README.md

```diff
@@ -1,229 +1,60 @@
-  sources:
-    - uri: https://file-examples.com/wp-content/uploads/2017/02/zip_2MB.zip
-      name: sample_2mb_file
-    - uri: https://file-examples.com/wp-content/uploads/2017/02/zip_5MB.zip
-      name: sample_5mb_file
-  # intervale in seconds at which the updater runs
-  update_interval_seconds: 300
-  # Should the downloaded files be used to create signatures in the system
-  generates_signatures: false
+```bash
+apt install jq
+pip install git+https://github.com/CybercentreCanada/assemblyline-service-template.git
+cruft create https://github.com/CybercentreCanada/assemblyline-service-template.git
 ```
 
-### Dockerfile
+## Documentation
 
-A Dockerfile is required to build the service container that will be executed in the system.
+For more information about service development for Assemblyline, follow this [guide](https://cybercentrecanada.github.io/assemblyline4_docs/developer_manual/services/developing_an_assemblyline_service/).
 
-The following items must be set for all services:
+---
 
-- All services must be based on the `cccs/assemblyline-v4-service-base:latest` image
-- An environment variable must be set for the service path
-- Install any service requirements
-- Copy the service code into `/opt/al/al_service/`
+# Assemblyline 4 - Service Base
 
-```dockerfile
-FROM cccs/assemblyline-v4-service-base:latest
+Ce référentiel fournit les fonctionnalités de base des services Assemblyline 4.
 
-# Set the service path
-ENV SERVICE_PATH result_sample.ResultSample
+## Créer un nouveau service Assemblyline
 
-# By default, the base service container as the assemblyline user as the running user
-#  switch to root to perform installation of dependancies
-USER root
+Vous pouvez créer un nouveau service Assemblyline en utilisant ce [template](https://github.com/CybercentreCanada/assemblyline-service-template).
 
-# See that we all these operations in one line to reduce
-#  the number of container layers and size of the container
-RUN apt-get update && apt-get install -y my_debian_apt_dependency_package && rm -rf /var/lib/apt/lists/*
-RUN pip install --no-cache-dir --user my_pip_dependency && rm -rf ~/.cache/pip
+## Variantes et étiquettes d'image
 
-# Change to the assemblyline user to make sure your service does not run as root
-USER assemblyline
+| **Type d'étiquette** | **Description**                                                                                                  |  **Exemple d'étiquette**   |
+| :------------------: | :--------------------------------------------------------------------------------------------------------------- | :------------------------: |
+|       dernière       | La version la plus récente (peut être instable).                                                                 |          `latest`          |
+|      build_type      | Le type de compilation utilisé. `dev` est la dernière version instable. `stable` est la dernière version stable. |     `stable` or `dev`      |
```

Comment:
build_type --> type de déploiement

Reply:
Hmm... should we fix that in our service template as well?
https://github.com/CybercentreCanada/assemblyline-service-template/tree/main/%7B%7B%20cookiecutter.__repository%20%7D%7D

Reply:
Probably ? I think it would be cleaner but it's just a suggestion